src/app/taquin/taquin.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, OnInit, OnChanges, Input } from '@angular/core';
+import {shuffle} from 'lodash';
+
+@Component({
+  selector: 'app-taquin',
+  templateUrl: './taquin.component.html',
+  styleUrls: ['./taquin.component.css']
+})
+
+
+
+export class TaquinComponent implements OnInit, OnChanges {
+
+  @Input() TilesNumber: number;
```

Comment:
Just a naming suggestion
```suggestion
  @Input() gridSize: number;
```